core/crdt/composite.go

```diff
@@ -119,12 +119,12 @@ func NewCompositeDAG(
 	}
 }
 
-// GetSchemaID returns the schema ID of the composite DAG CRDT.
+// ID returns the schema ID of the composite DAG CRDT.
 func (c CompositeDAG) ID() string {
 	return c.key.ToString()
 }
 
-// GetSchemaID returns the schema ID of the composite DAG CRDT.
+// Value returns the schema ID of the composite DAG CRDT.
```

Comment:
question: Same comments for `ID()` and `Value()`?

Reply:
The comments, and the `Value` function implementation make no real sense to me anyway, but I felt the need to at least tie the comments to the functions upon which they sit. I see further changes here as out of scope.